examples/templates/kubernetes/main.tf

```diff
@@ -106,6 +106,11 @@ resource "kubernetes_pod" "main" {
   metadata {
     name      = "coder-${lower(data.coder_workspace.me.owner)}-${lower(data.coder_workspace.me.name)}"
     namespace = var.namespace
+    labels = {
+      "app.kubernetes.io/name"     = "coder-workspace"
+      "app.kubernetes.io/instance" = "coder-workspace-${lower(data.coder_workspace.me.owner)}-${lower(data.coder_workspace.me.name)}"
+      "app.kubernetes.io/part-of"  = "coder"
```

Comment:
Should add some more labels similar to what we do in v1 for workspace details. Should add these to the PVC too.

Reply:
Added the labels I could from v1

```bash
$ kdpo coder-colin-gea
Name:             coder-colin-gea
Namespace:        colin-coder
Priority:         0
Service Account:  default
Node:             gke-dev-4-2-coder-node-pool-8b0aaa9c-wfg6/10.1.0.102
Start Time:       Wed, 18 Jan 2023 10:49:01 -0600
Labels:           app.kubernetes.io/instance=coder-workspace-colin-gea
                  app.kubernetes.io/name=coder-workspace
                  app.kubernetes.io/part-of=coder
                  com.coder.resource=true
                  com.coder.user=b851db54-9827-45bc-b453-72138b46f99c
                  com.coder.username=colin
                  com.coder.workspace.id=8118ca50-2edc-4b7c-8b70-61b3c7c3673d
                  com.coder.workspace.name=gea
```

```bash
$ kd pvc coder-colin-gea-home
Name:          coder-colin-gea-home
Namespace:     colin-coder
StorageClass:  standard
Status:        Bound
Volume:        pvc-d6612185-b553-459a-9c85-acce9bf33cc2
Labels:        app.kubernetes.io/instance=coder-pvc-colin-gea
               app.kubernetes.io/name=coder-pvc
               app.kubernetes.io/part-of=coder
               com.coder.resource=true
               com.coder.workspace.id=8118ca50-2edc-4b7c-8b70-61b3c7c3673d
               com.coder.workspace.name=gea
```